src/solvers/dgsem_tree/dg_1d.jl

```diff
@@ -284,7 +284,11 @@ end
     return nothing
 end
 
-@inline function calcflux_fv!(fstar1_L, fstar1_R, u::AbstractArray{<:Any, 3},
+# Compute the normal flux for the FV method on cartesian subcells, see
```

Comment:
"Cartesian" doesn't make sense in 1D, right? I don't think that we need to write something here, but if so, I would use "LGL subgrid" in some way.